src/index.js

```diff
@@ -85,11 +85,15 @@ export async function addObjectsToData(objects) {
   return merge({}, ...dataSets.concat(fileContents));
 }
 
-export async function renderHandlebarsTemplate(files, outputDirectory = process.cwd(), data = {}) {
+export async function renderHandlebarsTemplate(files, stdout = false, outputDirectory = process.cwd(), data = {}) {
```

Comment:
To keep backwards compatibility, it'd be great if this was the last argument, so:

```js
export async function renderHandlebarsTemplate(files, outputDirectory = process.cwd(), data = {}, stdout = false) {
```